### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by beteland123**
implement the following interactions:

   When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  When the user clicks (or taps) the close (X) button, the popup disappears.

- use a JavaScript array to store all of the information for all projects.

-update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. 
- implement the popup window.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -390,7 +198,108 @@ <h2 class="contact-prim" >Contact me</h2>
...
 </section>
+<div class="modal project-detail flex-column">
+        <div class="modal-wrap flex flex-column">
+          <div class="modal-header flex">
+            <svg
+              class="modal-close"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
```
Comment (OyePriscilla) [inline index.html:209]: - [ ] Kindly render the pop-up section dynamically with JS according to the project requirement. 👍

File: index.html [hunk 2]
```diff
@@ -390,7 +198,108 @@ <h2 class="contact-prim" >Contact me</h2>
...
+                  <div class="other-details">
+                    <div class="work-modal-languages">
+                      <ul class="work-categories"></ul>
+                    </div>
+                    <div class="flex modal-content-links">
+                      <a
+                        id="work-modal-live-demo"
+                        href=""
+                        class="button button-enabled flex"
+                        >See live
```
Comment (OyePriscilla) [inline index.html:246]: - [ ] Please kindly design the elements of this container (**for mobile version**) according to the given template.

> |     |Your Design                                 | Figma Design                                  |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     |  Untitled| Untitled
|

File: style.css [hunk 3]
```diff
@@ -277,229 +263,249 @@
...
+
+  .modal-wrap {
+    padding: 1rem;
+    background-color: #fff;
+    border: 1px solid #272729;
+    border-radius: 16px;
+    justify-content: flex-start;
+    overflow: scroll;
+    overflow-y: auto;
+  }
```
Comment (OyePriscilla) [inline style.css:434]: - [ ] Kindly correct this anomaly, I observed that the modal for the mobile version is not yet fixed on the UI, and this made other part of the app to be scrollable. 👍 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @beteland123 and @Xecyc 👏 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Populate project cards dynamically with JS ✔️ 
Use JavaScript to manipulate DOM elements ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (AdnanOlarmmi) [reply index.html:246]: > 

- [ ] Nice one trying to fix the design as requested by the previous reviewer, please, kindly note that this has created another bug, kindly fix this to match the Figma design above.

- Your design

image

- The left, bottom and top margins of technologies `html css javascript`.

- The overflow of the seelive and the border and icon of the see source buttons respectively.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @beteland123, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice one rendering the pop up dynamically.
- Nice one taking to the requested changes.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @beteland123 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

kpop-approved


## Highlights
:heavy_check_mark: Nice effort in making the requested changes
:heavy_check_mark: Used JavaScript to manipulate DOM elements 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**